README.md

```diff
@@ -0,0 +1,17 @@
+# OpenRPC Specification for the Filecoin Common Node API
+
+This is an accessory [this FIP](TODO(aatifsyed): add link)
+
+The main document is the [`spec.json`](./spec.json), which is a description of a
+set of [JSON-RPC](https://www.jsonrpc.org/) methods in [OpenRPC](https://spec.open-rpc.org/).
+
+Most of the functionality in [`tool`](./tool/) was used to generate the above
+document, but also of note is a `diff` function which can summarize differences
```

Comment:
Is diff a subcommand? Or a separate executable? 

Generally this reads a bit like a storybook, not technical docs

Reply:
I've tried to make the `tool --help` look more like technical docs, and the readme point to how to get there

Reply:
let me know what you think :)